### PR TITLE
Make image buildable

### PR DESCRIPTION
**Pull request by GeorgeBekh**
Right now it's not possible to build image from this dockerfile

This PR fixes that issue

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -326,7 +326,7 @@ RUN set -xe && \
 
 
 ENV FBC_VERSIONS \
-      1.05.0 
+      1.07.1
```
Comment (hermanzdosilovic) [inline Dockerfile]: Why did you change FBC version? This will break the Judge0 API version.

File: README.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ Here is a list of supported languages:
 |:---:|:---:|
 |1 |Bash (4.4)|
 |2 |Bash (4.0)|
-|3 |Basic (fbc 1.05.0)|
+|3 |Basic (fbc 1.07.1) |
```
Comment (hermanzdosilovic) [inline README.md]: Same as the previous comment. This cannot get into new release because it will break Judge0 API major version. This is essentially API change.

File: Dockerfile (hunk 1 above)
Comment (GeorgeBekh) [reply Dockerfile]: Because it's no longer possible to download version 1.05.0 from that sourceforge repo. If it didn't break the build i wouldn't change that :)

File: README.md (hunk 2 above)
Comment (GeorgeBekh) [reply README.md]: Well, it's a minor change isn't it? The id and the major version of the language are the same. We'll just have to update the seeds.rb file.

File: Dockerfile (hunk 1 above)
Comment (GeorgeBekh) [reply Dockerfile]: Ok, i found it in "older releases" directory. I'll use that

**Comment by GeorgeBekh** [comment]
> One more thing that is not in the comments. Can you please in this line inherit from judge0/buildpack-deps:buster-2019-11-23.

If i insert this line the build breaks on gcc 4.9

I suggest doing this update in another PR. It'll probably require a lot of changes

**Comment by GeorgeBekh** [comment]
@hermanzdosilovic I've left the same fbc version, but it works now. Also i've changed NodeJS installation. Now it's just downloads binaries

File: README.md [hunk 3]
```diff
@@ -16,51 +16,51 @@ We are open to any suggestions on how to reduce size of this image but retain fl
...
+|    17 | C# (mono 5.2.0.224)                  |
+|    18 | Clojure (1.8.0)                      |
+|    19 | Crystal (0.23.1)                     |
+|    20 | Elixir (1.5.1)                       |
+|    21 | Erlang (OTP 20.0)                    |
+|    22 | Go (1.9)                             |
+|    23 | Haskell (ghc 8.2.1)                  |
+|    24 | Haskell (ghc 8.0.2)                  |
+|    25 | Insect (5.0.0)                       |
+|    26 | Java (OpenJDK 9 with Eclipse OpenJ9) |
```
Comment (hermanzdosilovic) [inline README.md]: With the above update in Dockerfile, this now is just `OpenJDK 9` and not `OpenJDK 9 with Eclipse OpenJ9`.

**Comment by GeorgeBekh** [comment]
@hermanzdosilovic why've you changed the base branch?

**Comment by GeorgeBekh** [comment]
If you've already fixed the build please merge it to the master branch so i can add other languages e.g. PHP
